hera_cal/noise.py

```diff
@@ -51,8 +83,7 @@ def predict_noise_variance_from_autos(bl, data, dt=None, df=None, nsamples=None)
         Noise variance predicted on baseline bl in units of data squared
```

Comment:
Maybe mention the default value for uniform time samples just so people easily understand where things come from.

Reply:
Done.